tsdb/querier_test.go

```diff
@@ -867,7 +867,7 @@ func TestPopulateWithDelSeriesIterator_DoubleSeek(t *testing.T) {
 	testutil.Equals(t, float64(2), v)
 }
 
-// Regression when seeked chunks were still found via binary search and we always
+// Regression when sought chunks were still found via binary search and we always
```

Comment:
This was right.

Reply:
sure? https://www.oxfordlearnersdictionaries.com/definition/english/seek